cmd/podman/varlink.go

```diff
@@ -52,7 +54,7 @@ func varlinkCmd(c *cli.Context) error {
 	}
 
 	// Run the varlink server at the given address
-	if err = service.Listen(args[0], 0); err != nil {
+	if err = service.Listen(args[0], 10*time.Second); err != nil {
```

Comment:
Lets add an --timeout option and allow to me specified in microseconds.

Reply:
what do you want the default to be then ?